src/VirtualClient/VirtualClient.UnitTests/VirtualClient.UnitTests.csproj

```diff
@@ -62,6 +62,9 @@
 		<None Update="TestData\TarGZDummyArchive.tar.gz">
 			<CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
 		</None>
+        <None Update="TestData\TestWrathmark.txt">
```

Comment:
Test data should go to VirtualClient.Actions.UnitTests/Examples, so that you don't need this here and you don't need to change the assemblyInfo